File: c++/literals_constants_variables/variables.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>
#include <string>

using std::string;
using std::cout;
using std::endl;

int main()
{
  string name;
  int quantity;

  name = "C++";
  quantity = 5;

  cout << "I am the " << name
    << " programmer with " << quantity
    << " years of experience." << endl;
}

File: pascal/literals_constants_variables/variables.pas
(*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 *)

program Variables;

var
  name: string;
  quantity: integer;

begin
  name := 'Pascal';
  quantity := 5;

  writeln(
    'I am the ', name,
    ' programmer with ', quantity,
    ' years of experience.'
  );
end.
